packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -891,20 +891,31 @@ export class ConnectionManager implements IConnectionManager {
 
         this._outbound.push(messages);
     }
-
+    private count: number = 0;
+    private readonly ops: any[] = [];
```

Comment:
I'm not sure what is the logic here:
- ops is not really used. You push ops into it, but never pull or access it in any other way
- `count` is too generic name and should be renamed to reflect the purpose of this counter.